components/events/Event.tsx

```diff
@@ -0,0 +1,134 @@
+import {
+  AccordionButton,
+  AccordionItem,
+  AccordionPanel,
+  Flex,
+  Icon,
+} from "@chakra-ui/react";
+import { useTrack } from "hooks/analytics/useTrack";
+import type { FC } from "react";
+import { FaMapMarkerAlt } from "react-icons/fa";
+import { FiCalendar, FiClock } from "react-icons/fi";
+import { Badge, Heading, LinkButton, Text } from "tw-components";
+
+interface EventProps {
+  type: string;
+  title: string;
+  timestamp: string;
+  location: string;
+  description: string;
+  link: string;
+}
+
+const Event: FC<EventProps> = ({
+  type,
+  title,
+  timestamp,
+  location,
+  description,
+  link,
+}) => {
+  const trackEvent = useTrack();
+
+  return (
+    <AccordionItem
+      bg="whiteAlpha.100"
+      py={10}
+      px={{
+        base: 2,
+        md: 10,
+      }}
+      my={4}
+    >
+      <AccordionButton
+        gap={6}
+        _hover={{
+          bg: "transparent",
+        }}
+        display="flex"
+        flexDir={{
+          base: "column",
+          md: "row",
+        }}
+      >
+        <Badge
+          h="min"
+          alignSelf={{
+            base: "flex-start",
+          }}
+        >
+          {type}
+        </Badge>
+        <Flex flexDir="column" w="full">
+          <Heading textAlign="left" size="title.md">
+            {title}
+          </Heading>
+
+          <Flex justify="space-between" mt={2}>
```

Comment:
This spacing looks a bit weird, try it with a gap between the items and justify at the start.